lib/Dist/Zilla/Plugin/IAChangelog.pm

```diff
@@ -73,14 +73,18 @@ sub gather_files {
                 $id = 'cheat_sheets';
             }
             else{
-                if(my $ia = DDG::Meta::Data->get_ia(module => $m)){
-                    unless(@$ia == 1){
-                        $s->log_fatal(["Multiple IDs in metadata for module $m: " . join(', ', map { $_->{id} } @$ia)]); 
+                while($m =~ /::/){
+                    if(my $ia = DDG::Meta::Data->get_ia(module => $m)){
+                        unless(@$ia == 1){
+                            $s->log_fatal(["Multiple IDs in metadata for module $m: " . join(', ', map { $_->{id} } @$ia)]); 
```

Comment:
Yes, cheat sheets, for example.  This is to catch cases where the same module name is used or we get another case like cheat sheets we don't know about.